Test/test_bank_acc.py

```diff
@@ -1,69 +1,64 @@
 import unittest
-import BankAcc
+from Challenges import bank_account
```

Comment:
As discussed, since you are accessing only one class from bank_account module, you can do import this way
`from challenges.bank_account import BankAccount`